docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,10 @@ IBM® block storage CSI driver 1.10.0 introduces the enhancements that are detai
 
 **General availability date:** 15 July 2022
 
+## New automatic host connectivity definition
+
+IBM® block storage CSI driver 1.10.0 now supports automatic host definition for new host connectivity.
```

Comment:
@matancarmeli7 what does `for new host connectivity` mean? if we add a new connectivity for an existing host, we won't update its definition on storage

Reply:
to clarify - can update to "for new storage host connectivity"

Reply:
if by `new storage host connectivity` you mean new ports for an existing host definition on storage, that's not supported here either

Reply:
updated

Reply:
what does `newly defined storage host connectivity` mean? new type of connectivity? because I don't think we change old existing host definitions when workers change their connectivity type (NVMe/FC/iSCSI)

Reply:
@oriyarde what exactly **does** it include once its installed?

Reply:
dynamic host definition for new nodes (the connectivity is a detail of the definition, the trigger of the feature is when adding nodes)

Reply:
but perhaps @matancarmeli7 could clarify better

Reply:
it can be more than a sentence long. but we need to say what this feature does and why it should be used

Reply:
same as in volume provisioning - "dynamic" means the user doesn't needed to statically define the "dynamic object" on storage in advance

Reply:
updated

Reply:
opening a new conversation for current revision